.gitignore

```diff
@@ -1,4 +1,3 @@
-dist/
```

Comment:
Hi there! thanks for taking this, do you know why should we save the dist files? they are auto generated when it is built

Reply:
ah oops, this change got into this PR too... I didn't mean to.

What happened was... I couldn't wait for this PR to merge into sass-export because I need to use this fix immediately in my work... So I referenced my fork as a dependency instead of the official sass-export... but since it's just a git project (and not an npm module), I have to update the generated dist files in order to use the updated changes...?

In any case, I didn't realize my commit would show up here too...

I'll probably close this PR and re-open another PR with just the previous commit...

Reply:
cool! make sense, thank you!